src/services/WalletService/deterministic/ledger.ts

```diff
@@ -109,8 +109,7 @@ export class LedgerWallet extends HardwareWallet {
       const msgHex = Buffer.from(msg).toString('hex');
       const ethApp = await makeApp();
       const signed = await ethApp.signPersonalMessage(this.getPath(), msgHex);
-      // @ts-ignore
-      const combined = addHexPrefix(signed.r + signed.s + signed.v.toString(16));
+      const combined = addHexPrefix(signed.r + signed.s + signed.v);
```

Comment:
The original `toString()` seemed like an error.

Reply:
I don't think it is. `v` is a regular number, not a hexadecimal string by default. In the signature it should be hexadecimal.

Reply:
The `.d.ts` file types it as string. https://github.com/MyCryptoHQ/MyCrypto/blob/265677604311c58cfdfbf97933112811e7cf6e2f/src/types/vendor/ledger/hw-app-eth/index.d.ts#L6

But if I change the type, other conflicts arise.
There is a mismatch in types and usage.
Since this is out of scope I reverted the change and added a todo comment.